### ISSUE TITLE
Stylelint CLI produces invisible text with certain color schemes need a way to defect "colorization"

### ISSUE THREAD

**Issue by richtong** [stylelint/stylelint#1822]
Ugh this is a tricky one, so I don't know if you should document or not, but with certain Ubuntu 14.04 color schemes (like solarized), the line number and column positions are invisible. The color scheme chosen for style lint's output is completely black

> Which rule, if any, is this issue related to?

Is there a way to turn off the coloring of output

> What CSS is needed to reproduce this issue?

Add the following into a file

```css
.foo {
  fsdf
  color: pink;
}
```

And here is the output:

```
foo.css
 2:5  ✖  Expected indentation of 2 spaces   ind
```

But if you go to Ubuntu (should I add a screen shot, it is invisible..
![untitled](https://cloud.githubusercontent.com/assets/1782087/17817331/d5a7a4d4-65f2-11e6-947b-6d227b944796.png)

> What stylelint configuration is needed to reproduce this issue?

This uses the stylelint-default-config

> Which version of stylelint are you using?

7.1.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

 `stylelint foo.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

Visible line numbers 

> What actually happened (e.g. what warnings or errors you are getting)?

Do not get warnings but good to have a non colorized output

**Comment by evilebottnawi** [comment]
@richtong use `--no-color`

**Comment by richtong** [comment]
Got it thanks. Would be great to document on the rradme page. 

Thanks,
Rich

No! Try not. Do, or do not. There is no try. Master Yoda. Star Wars Episode V: The Empire Strikes Back

http://tongfamily.com | @richtong | http://fb.me/richtong1 | +1-206-963-9029

**Comment by evilebottnawi** [comment]
@richtong we have `--help` arguments in CLI.

**Comment by jeddy3** [comment]
Which is, in turn, mentioned in the [CLI userguide](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/cli.md).

I think we've got this one covered, so I'll close the issue.

**Comment by richtong** [comment]
Thanks yes. 

Thanks,
Rich

No! Try not. Do, or do not. There is no try. Master Yoda. Star Wars Episode V: The Empire Strikes Back

http://tongfamily.com | @richtong | http://fb.me/richtong1 | +1-206-963-9029
